src/Windowing/OpenToolkit.Windowing.Desktop/NativeWindow.cs

```diff
@@ -42,6 +42,10 @@ public class NativeWindow : INativeWindow
 
         private KeyboardState _keyboardState = default;
 
+        // Current cursor we assigned to the GLFW window.
+        // Null if the cursor is default.
```

Comment:
Later on we document this as "cursor cannot be null". Even though there's a good reason why, it does read as a little inconsistent.

Any ideas on how to resolve this?